cves/CVE-2013-0856.yml

```diff
@@ -64,7 +64,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  In the code that handles the crafting of Apple Lossless Audio Codec, there was a
```

Comment:
What is Apple Lossless Audio Codec? Maybe explain this in a bit more detail

Reply:
Apple lossless codec is a way of encoding audio for apple devices. Will add and fix.